langchain/output_parsers/pydantic.py

```diff
@@ -30,10 +30,11 @@ def get_format_instructions(self) -> str:
         schema = self.pydantic_object.schema()
 
         # Remove extraneous fields.
```

Comment:
noting @hwchase17 : added this back to support more complex types, like an enum.. I tweaked the in-context example as a result, including a positive + negative case. Seems to be working reasonably well